azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -123,7 +122,7 @@ func resourceArmMsSqlDatabase() *schema.Resource {
 				Type:         schema.TypeFloat,
 				Optional:     true,
 				Computed:     true,
-				ValidateFunc: azValidate.FloatInSlice([]float64{0.5, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 3, 4, 5}),
+				ValidateFunc: validation.FloatAtLeast(0),
```

Comment:
alternatively we could add `0` as an allowed value here:

```suggestion
				ValidateFunc: azValidate.FloatInSlice([]float64{0, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 3, 4, 5}),
```

can we also add an acceptance test for this SKU?